src/modules/weather/WeatherData.cpp

```diff
@@ -1,16 +1,15 @@
-#pragma once
-
-#include <string>
-#include <vector>
+#include "WeatherData.h"
```

Comment:
This C++ source file should be named WeatherData.cpp and should not be in data directory, but next to Weather.cpp file